src/containers/AsideNavigation/AsideNavigation.tsx

```diff
@@ -62,26 +65,50 @@ export interface AsideNavigationProps {
 
 enum Panel {
     UserSettings = 'UserSettings',
-}
-
-function getDocumentationLink() {
-    // Use saved language from settings if it's present, otherwise use browser language
-    const lang = settingsManager.readUserSettingsValue(LANGUAGE_KEY, navigator.language);
-
-    if (lang === 'ru') {
-        return 'https://ydb.tech/docs/ru/';
-    }
-
-    return 'https://ydb.tech/docs/en/';
+    Information = 'Information',
+    Hotkeys = 'Hotkeys',
 }
 
 export function AsideNavigation(props: AsideNavigationProps) {
```

Comment:
Please, don't forget to make similar changes in internal version, there is a different navigation component there